src/main.cpp

```diff
@@ -206,10 +206,11 @@ int wmain(int argc, wchar_t* argv[])
                     return E_FAIL;
                 }
                 noAdminSpecified = true;
+                noCpuSamplingSpecified = true;
```

Comment:
I don't think it's necessary to set the `noCpuSamplingSpecified` variable here since the `DoStart` function turns off CPU samples already when admin is specified (it sets the system flags to 0, which removes CPU samples). This would avoid adding the special case logic below for the CPU sampling error message.

Reply:
I tested this and you're right. I'll remove this